omx/gstomxaacdec.c

```diff
@@ -62,10 +64,10 @@ gst_omx_aac_dec_class_init (GstOMXAACDecClass * klass)
       GST_DEBUG_FUNCPTR (gst_omx_aac_dec_get_channel_positions);
 
   audiodec_class->cdata.default_sink_template_caps = "audio/mpeg, "
-      "mpegversion=(int){2, 4}, "
-      "stream-format=(string) { raw, adts, adif, loas }, "
+      "mpegversion=(int){ 2, 4 }, "
+      "stream-format=(string) { raw, adts, adif }, "
       "rate=(int)[8000,48000], "
-      "channels=(int)[1,9], " "framed=(boolean) true";
+      "channels=(int)[1,2], " "framed=(boolean) true";
```

Comment:
The AAC encoder for dm81xx support up to 5 channels, does the decoder support only 2 channels?

Reply:
Yes, the aac decoder supports only standard stereo 1/2 channels.